frontend/src/components/common/Terminal.tsx

```diff
@@ -384,7 +404,11 @@ export default function Terminal(props: TerminalProps) {
       }}
       keepMounted
       withFullScreen
-      title={t('Terminal: {{ itemName }}', { itemName: item.metadata.name })}
+      title={
+        isAttach
+          ? t('Attach : {{ itemName }}', { itemName: item.metadata.name })
```

Comment:
Remove space between `Attach` and `:`, so it's consistent with the other title.

Reply:
updated